modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -550,6 +717,116 @@ private void _assertGetObjectDefinitionsPageWithFilter(
 			expectedObjectDefinitions, (List<ObjectDefinition>)page.getItems());
 	}
 
+	private boolean _equals(
+		ObjectValidationRule objectValidationRule1,
+		ObjectValidationRule objectValidationRule2) {
+
+		if (objectValidationRule1 == objectValidationRule2) {
+			return true;
+		}
+
+		if (!Objects.deepEquals(
```

Comment:
@pedro-oliveira446 the `BaseObjectValidationRuleResourceTestCase` already has a 
```
protected boolean equals(
	ObjectValidationRule objectValidationRule1,
	ObjectValidationRule objectValidationRule2) {
```
method. So it makes more sense to move this test to `ObjectValidationRuleResourceTest` because we are testing the `ObjectValidationRule`. 

This way we can reuse the existing method.

Reply:
As we talked, I tried a new approach in the same class.